utils/scripts/gen_caffe2_model.py

```diff
@@ -0,0 +1,58 @@
+## This is a helper script that generates simple Caffe2 models.
+## The generated model will be used for Caffe2 importer unittest:    
+## ./tests/unittests/caffe2ImporterTest.cpp 
+## Run $>python gen_caffe2_model.py to get the model files.
```

Comment:
Oh I realize now, that the model we want to generate is actually hardcoded in the script ><.
Could you say in the first sentence that the issued model is currently a simple convolution network.

Also could you add a comment to explain what simple means in this context?
For instance, I don't see what are the limitations in the graphs we can write at first glance.

Reply:
Since all ops and weights are added manually in the scripts, I don't think we would like to use this script to generate models with a lot of layers or with the input using the real image data. Therefore, I just put the "simple" here.  You are right,  there is no limitations in the graph so far:)  So I think it is better to remove "simple" rather than why it is "simple".